CoEpi/ui/RootWireframe.swift

```diff
@@ -48,9 +48,18 @@ extension RootWireFrame : HomeViewModelDelegate {
     }
     
     private func showQuiz() {
-        let quizViewController = HealthQuizViewController.init(nibName: String(describing:HealthQuizViewController.self), bundle: nil
-        )
+        let viewModel: HealthQuizViewModel = try! container.resolve()
+        
+        let quizViewController = HealthQuizViewController(viewModel: viewModel)
+
         quizViewController.title = "My Health"
```

Comment:
Maybe move this to the view controller? Don't see a reason to set it from outside.